python/lsst/ip/isr/defects.py

```diff
@@ -513,6 +611,13 @@ def _get_values(values, n=1):
 
         return values[:n]
 
+    def getReasonDict(self):
```

Comment:
I would prefer if there were too methods here ... `getReasonToBitMapping()` and `getBitToReasonMapping()` or something like that?
Also this doesn't have a docstring.